lib/internal/Magento/Framework/Component/ComponentRegistrar.php

```diff
@@ -51,7 +51,7 @@ public static function register($type, $componentName, $path)
                 . 'has been already defined in \'' . self::$paths[$type][$componentName] . '\'.'
             );
         } else {
-            self::$paths[$type][$componentName] = str_replace('\\', '/', $path);
+            self::$paths[$type][$componentName] = realpath(str_replace('\\', '/', $path));
```

Comment:
We must not operate with absolute file paths.

@navarr could you provide details of `BUNDLE-1798` please?